### ISSUE TITLE
label montage - head/tail points only set in CPR parameters 

### ISSUE THREAD

**Issue by arobie** [kristinbranson/APT#329]
If you haven't chosen head/tail points in CPR parameters then  Label > Label Montage Overlay (trx centered) head/tail aligned plot defaults to pts 1 and 2. 

Maybe this should be under the non-trx aligned option as well/instead? I assume in  freely moving single animals projects this would be helpful. 

Can it pop-up a warning if you haven't set head/tail points?

**Comment by allenleetc** [comment]
Yes agreed head/tail aligned would be useful for non-trx projs as well eg with center-of-mass centering or similar.

Regarding how the head/tail landmarks are specified. Yes the head/tail spec seems more general than a Tracking Parameter (and certainly more general than a CPR-specific tracking parameter). Maybe specification with the Skeleton or in that vicinity makes more sense. For instance you might want to give every landmark a text label (Nose, LeftEar, etc); this has been requested previously. We could piggyback Head/Tail specification onto such a UI. (Maybe this is the same as the skeleton UI but beefed up.)

The head/tail spec could then be used for alignments (eg labels, or tracking results), as well as simple metrics like azimuth/phi which are pretty general
